week2/homework/src/index.js

```diff
@@ -1,3 +1,59 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+var fs = require('fs')
+let filename = './todolist.txt';
+let currentList = fs.readFileSync(filename, "utf-8").toString().split("\n");
+let whichFunction = process.argv[2];
+let rawInput = process.argv.splice(3).join(' ')
+let userInput = `${rawInput}`
```

Comment:
The `rawInput` variable declared on line 7 is only used on line 8. Why do we need to declare this variable at all?